### DISCUSSION TITLE
Release of Heap-Use-After-Free Read fix

### DISCUSSION THREAD

**Discussion by Staerkjaer** [qpdf/qpdf#1150] [General]
Hi qpdf team 👋

First of all thank you for this tool and the work you put into it!

I'm in the works of integrating qpdf into my project, but i noticed that the lib has a vulnerability [OSV-2024-18](https://osv.dev/vulnerability/OSV-2024-18). As far as i can tell, a fix has already been [committed](https://github.com/qpdf/qpdf/compare/0109e365decfdc2e8fbb920b4179d32daeeb96ae...6b80e0f14b296c21d38a92e25af72da9bf5757ae), so i was wondering if you could tell me when the next planned release is and if you plan on including the fix in said release.

**Comment by jberkenbilt** [comment]
I plan on release 11.9.0 in the next week or so with the fix.

**Reply by Staerkjaer** [reply]
Great, thank you for your fast reply!

**Reply by jberkenbilt** [reply]
I should mention that, in response to that issue, I added fuzzing tests to JSON (where the problem was) and found and fixed several other issues with the JSON parser, though none beyond this were of this nature.
